### DISCUSSION TITLE
Room for improvement

### DISCUSSION THREAD

**Discussion by proyb6** [tabler/tabler-icons#385] [General]
Glad that you have release 3000th icons before the end of 2022. But there are more improvement I suppose web devs love "hyped" in technology and design which seem to led to more adoptions. My suggestion is that Tablet site could be revamp as least as exciting as Font Awesome. These including able to copy code for Vue 3, Angular, Svelte or even more customizable?

Most devs does not know Tabler, there is a need for comparison page.

**Comment by codecalm** [comment]
Hi  @proyb6! Thank you for nice words :) 

I'm working on v2.0 of Tabler Icons and new website :) In v2.0 will be available new packages like Vue, React, Preact, Svelte, solid.js, Angular and few more. New website with cool documentation also will be recreated :)

https://github.com/tabler/tabler-icons/pull/378

**Reply by proyb6** [reply]
My suggestion would be ask the community for inputs and votes so we get everyone who cares about to participate and brings a better release.
